app/jobs/dradis/plugins/projects/project_clone_job.rb

```diff
@@ -0,0 +1,47 @@
+module Dradis::Plugins::Projects
+  class ProjectCloneJob < ApplicationJob
+    queue_as :dradis_project
+
+    def perform(project_id:, new_project_id:)
+      @new_project = Project.find_by_id(new_project_id)
+      @project = Project.find_by_id(project_id)
+
+      @lookup_table = template_importer.parse(template_xml)
```

Comment:
Note that the return of this has a FIXME comment. Instead of relying on something that might be fixed later, can we instead get the `lookup_table` from the importer class using a getter?